redfish/provider/common.go

```diff
@@ -183,25 +194,25 @@ func NewConfig(pconfig *redfishProvider, rserver *[]models.RedfishServer) (*gofi
 		return nil, fmt.Errorf("no provider block was found")
 	}
 
-	// first redfish server block
-	if len(*rserver) == 0 {
```

Comment:
This check is required, rt? We do still want to throw an error if user doesn't provide the `redfish_server` block.

Reply:
hi @rounak-adhikary there are two same check. I removed one of them. Please help confirm the double check is necessary.

Reply:
Oops. The missed the extra check above.
U are rt. The double check isn't necessary. Thanks for removing it.